### PR TITLE
Send error response when ainize fails

### PR DESCRIPTION
**Pull request by minsulee2**
Now send error response if ainize fails to write on blockchain.

### CODE REVIEW

File: handlers/ainizeUtils.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
+const { ainizeAdmin } = require('../ainize');
```
Comment (platfowner) [inline handlers/ainizeUtils.js:1]: minor) how about using 'helper' like ainizeHelper or just handlerUtils?

**Review by platfowner — APPROVED** [review]
LGTM with some minor comments.
Thanks!

File: handlers/ainizeUtils.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+const { REST_MODE } = require('../env');
+
+class AinizeUtils {
+  static getDataFromServiceRequest = (req) => {
+    const request = ainizeAdmin.internal.getDataFromServiceRequest(req);
+    return request?.requestData;
```
Comment (jiyoung-an) [inline handlers/ainizeUtils.js:7]: did balance error occur because it wasn't parsed?

**Review by jiyoung-an — APPROVED** [review]
LGTM with some question, Thanks!

File: handlers/ainizeUtils.js (hunk 1 above)
Comment (minsulee2) [reply handlers/ainizeUtils.js]: I took ainizeHelper! Thanks !!

File: handlers/ainizeUtils.js (hunk 2 above)
Comment (minsulee2) [reply handlers/ainizeUtils.js:7]: This is discussed offline. Thanks for asking this. resolved.

**Pull request merged**